### DISCUSSION TITLE
Zone Expander with Line Level Outputs

### DISCUSSION THREAD

**Discussion by pseudotronics** [micro-nova/AmpliPi#155] [Ideas] [answered]
So far AmpliPi is everything I am looking for, the one thing that is missing for me is being able to provide line level audio to external amplifiers. Anyone have the same issue as me or would like to see an expander unit with RCA outputs for the zones?

**Comment by Lohrer** [comment, marked as answer]
AmpliPi does in fact provide line-level outputs for each zone. If you look at the [Quick Start Guide](https://github.com/micro-nova/AmpliPi/blob/master/docs/QUICK_START.md), look for PREOUT. Each zone has an amplified output and a line-level output that mirror each other, just with different output levels. Is that what you are looking for?

**Reply by linknum23** [reply]
I think the issue is that the specifications on our [website](https://www.micro-nova.com/amplipi) have not been updated since around the start of the Kickstarter. There have been several additions since then. Here are some of the highlights from our last design update on [Kickstarter](https://www.kickstarter.com/projects/micro-nova/amplipi-home-audio-system/posts/3141418):

- Added additional ports: 6x stereo pre-outs, optical in, and aux in. This new back panel is jam-packed! 
- Improved airflow, with two fans blowing across the internals to a large exhaust grille on the side of the case. (Fan behavior is configurable. They default to "silent" mode, only spinning up if necessary.)
- Improved upgradability with separate removable panel for the controller board and pre-amp board ports. This will give us the flexibility to offer upgrade kits that not only upgrade the boards, but also modify the I/O (much the same way that a PC case has a separate I/O panel for the motherboard.)

And from another post [update](https://www.kickstarter.com/projects/micro-nova/amplipi-home-audio-system/posts/3201665)

- Upgraded the OLED screen to a full color LCD touchscreen! We decided the touch feature really added a lot to the front panel interface - it allows for easy scrolling through status pages. The full color LCD and higher resolution is very nice as well - it provides a much richer status reporting screen.

That should get you up to date on the AmpliPi specs.

We are sorry about that, we will plan on updating our site soon.

**Reply by linknum23** [reply]
I've added #159 to keep track of the changes we need to make to get everything synced up.

**Comment by yoggiland** [comment]
Hi,

I am still waiting for my AmpliPi so I have no way of testing yet (also do to lack of knowledge on how things work)... I have a question I hope I could get answered here.

I would like to add an extra pair of speakers to the pre-out (through active speaker or an extra amp) so my question is regarding volume control.

Can volume be controlled by the signal coming from the pre-out or is the signal always at the same level regarding the volume out on the AmpliPi Chanel?

Or do the volume of the pre-out follow the volume of the channel output?

Joachim

**Reply by Lohrer** [reply]
Hello Joachim,

The volume setting of the amplified outputs also adjusts the volume of the preouts.

**Reply by bs42** [reply]
Is it possible in software to make a preout fixed volume and not follow the channel output volume?

I've got a subwoofer amplifier for a couple of in-ground outdoor subs that I'd like to run off of the pre-out of one of the sets of speakers I have in the back yard, but have the volume controlled by the sub amp and not the AmpliPi.

**Reply by Lohrer** [reply]
If you don't plan to use the amplified zone output, then you can just set the volume to max and leave it untouched in AmpliPi. If you are trying to use both the preout AND amplified outputs of a zone unfortunately it's the same signal, just one is amplified. See [the signal diagram](https://github.com/micro-nova/AmpliPi/blob/develop/hw/diagrams/signal_diagram.drawio.svg) for an idea of the prout vs amplified output split. It doesn't show the volume controllers, but there are 6 of them that sit right after the "source selectors".

**Reply by bs42** [reply]
Yeah, my intent was to use both since the one is simply the sub and would always be the same audio anyway.  I had seen that diagram and when it said "Line Level" & Speaker Output I took that to mean an fixed level output on the RCA as that is how that is typically described.

**Reply by Lohrer** [reply]
Hmm, I've tried to think of a way to make that setup work but since the output of the Preamp Board already has the volume attenuation applied the only thing I can think of would be to replace the cable going from the Preout Board to the Amp Board with a couple wires soldered to the Preamp Board bypassing the volume controllers and connecting straight to the Preout Board.

**Reply by bs42** [reply]
Got it, I'll figure a way to make it work.  Thanks for looking into it.

**Comment by yoggiland** [comment]
Hi Lohrer,

This is good news, thanks for the quick answer!

I want to be able to play music on the terrass a few times a year (when the weather permits it) but I don't want to dedicate an amplipi channel to it.

Now I need to find a suitable amplifier for my outdoor speakers, maybe something like this?

FX AUDIO M-DIY-MINI7498 Class D Amplifier Module TDA7498 2x50W 8 Ohm
https://www.audiophonics.fr/en/amplifier-boards/fx-audio-m-diy-mini7498-class-d-amplifier-module-tda7498-2x50w-8-ohm-p-12224.html

Do you know what amplifier chip is being used in the amplipi (not sure if it would matter to try and get a amp with the same chip)?

Joachim

**Reply by Lohrer** [reply]
We're using TDA7492E amplifiers, but it shouldn't matter what you use. I'm not sure what you're planning on using the associated amplified output for, but keep in mind there are only 6 stereo audio outputs per AmpliPi unit. So zone 1's audio will play out the amplified output 1 and preout 1 at the same time.
